### PR TITLE
Issue 12: Update site meta data to include the site banner

### PR DESCRIPTION
**Pull request by tacensi**
#12 Update site metadata to include site banner

I made two images folowing Twitter and Facebook sizes and included Gimp files. Notes:

- Removed antialias when scaled fighter
- Included a JS FIGHT CLUB! with the same font as the site's. Not actually a logo, but...

Both are easy to edit in Gimp.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -20,6 +20,20 @@
   <link href="./lib/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
   <link href="//fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
   <link href="//fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
+
+  <meta property="og:title" content="Javascrip Fight Club" />
```
Comment (Asuza) [inline public/index.html]: There is a minor typo here, but otherwise this PR looks great!

**Comment by Asuza** [comment]
I love what you did with the images! After testing it, there was only one minor issue. As soon as you have time to fix that, I will get this merged.

**Comment by tacensi** [comment]
Please don't hate me ;-)

I am sorry, I didn't notice it until I pushed it, and I also didn't know the updates to my branch would appear here automatically. :-S

I was also going to ask: since I am not 100% satisfied with the titles on the images, is it ok to try to make a logo with the shield used in the favicon? I didn't try it yet, I don't know if I scale the shield too much if it would appear nicely, but I gonna give a try.

**Comment by Asuza** [comment]
No worries, @tacensi! I've gotten into a habit of checking my `git status` and `git diff` before nearly every commit because I would do things like that as well. :smile:

In regards to the shield favicon, I don't want the existing one to be used in the image because that icon was inherited from the old repository, and a lot of sprites in the original version of the game were taken from other projects or games. I need a new, free/original icon to replace it.

That said, I don't don't mind if you want to play around with it more as long as you use a shield you created yourself or you found an appropriate open-source shield to use.

**Comment by Asuza** [comment]
@tacensi do you mind if I merge this (and you can improve on it in another PR), or would you prefer I wait?

**Comment by tacensi** [comment]
Yes, @Asuza, you can go ahead and merge.

I will look into the the shield and send another PR.

**Comment by Asuza** [comment]
Thanks, @tacensi. I hope to see more PRs from you soon!

**Pull request merged**